app/people_controller.rb

```diff
@@ -4,9 +4,23 @@ def initialize(params)
   end
 
   def normalize
+    wanted_values = [ 'first_name', 'city', 'birthdate' ]
+    merged_hash_content = [
+      data({ file_content: @params[:dollar_format] }),
+      data({ file_content: @params[:percent_format], separator: "%" }),
+    ]
+
+    merged_hash_content.flatten.sort_by!{ |obj| obj[@params[:order].to_s] }.map do |obj|
+      obj.values_at(*wanted_values).join(", ")
+    end
```

Comment:
I think it would be best if you put the logic inside this method onto a service object, then just make one call from here.

Then make `wanted_values` a constant of that class

And perhaps make line 13 to 15 more readable by moving some code into private methods

Reply:
I like your idea.